presidio-cli/presidio_cli/__init__.py

```diff
@@ -1,6 +1,10 @@
 """A Python CLI for analyzing PII Entities with Microsoft Presidio framework."""
 
+import importlib.metadata
+
+__version__ = importlib.metadata.version("presidio-cli")
```

Comment:
Will this work if the package is not yet installed?

Reply:
You need to setup poetry env anyway to run, so it installs anyway from the toml

Reply:
Maybe we can wrap this with try except? If someone is working on this codebase, there shouldn't be a strict requirement to install the package first. WDYT?

Reply:
done